### ISSUE TITLE
 Kickstart UI: Inaccessible links in tooltips

### ISSUE THREAD

**Issue by garrying** [vmware-tanzu/tanzu-framework#1174]
**Bug description**
- Tooltips containing in-line links to other pages are not clickable
- Lack of color contrast for in-line links in tool-tips. Example:
![Screen Shot 2021-11-16 at 11 59 19 AM](https://user-images.githubusercontent.com/227587/142030522-de4f4805-0ec1-4bf7-87b7-e903be8c10ed.png)
- There might be other instances, beyond the Amazon EC2 deploy flow

**Affected product area (please put an X in all that apply)**

- [x] User Experience

**Expected behavior**
- Based on the [Clarity examples](https://clarity.design/angular-components/tooltip/#usage), tooltips should just contain a concise string
- If a link is necessary, user should be able to keep the tooltip invoked by clicking on the tooltip (see Clarity page for example behaviour) 

**Steps to reproduce the bug**
- `tanzu management-cluster create --ui`
- Navigate to `http://127.0.0.1:8080/#/ui/aws/wizard`
- See tooltip at `AWS CREDENTIAL PROFILE` in Step 1

**Version** (include the SHA if the version is not obvious)

**Environment where the bug was observed (cloud, OS, etc)**

**Relevant Debug Output (Logs, manifests, etc)**

**Comment by miclettej** [comment]
Hi @stmcginnis could you please help move this over to tanzu-framework. I think someone outside the UI team might have added a link in one of the tooltips, which does not work as mentioned in this issue. I'll make an adjustment to the tooltip content. Thanks!

**Comment by miclettej** [comment]
@garrying thanks for opening this, looks like your first tanzu-framework issue. Appreciate your contribution. 

@swalner-vmware could you please add this to our GH project. I am unable to do so since I am temporarily removed from the TF organization.
